pkgs/applications/science/electronics/kicost/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, python3Packages, fetchFromGitHub }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "kicost";
+  version = "1.1.6";
+
+  src = fetchFromGitHub {
+    owner = "hildogjr";
+    repo = "KiCost";
+    rev = "v${version}";
+    sha256 = "kFuANwdEtGvPltTeC1vY1eWM7TY/3k4GyzdAG/lZLuo=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    beautifulsoup4
+    lxml
+    XlsxWriter
+    tqdm
+    requests
+    validators
+    wxPython_4_1
+    colorama
+  ];
+
+  doCheck = false;
```

Comment:
We have tests https://github.com/hildogjr/KiCost/tree/master/tests

Reply:
Done